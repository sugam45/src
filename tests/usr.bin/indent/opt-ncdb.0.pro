/* $NetBSD: opt-ncdb.0.pro,v 1.1 2021/03/06 17:56:34 rillig Exp $ */
/* $FreeBSD$ */

-ncdb
