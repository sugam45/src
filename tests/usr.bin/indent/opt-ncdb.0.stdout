/* $NetBSD: opt-ncdb.0.stdout,v 1.2 2021/03/06 23:09:17 rillig Exp $ */
/* $FreeBSD$ */

/* A single-line comment. */

/* A multi-line comment. */

/* A multi-line comment. */

int		ga;		/* A single-line comment. */

int		gb;		/* A multi-line comment. */

int		gc;		/* A multi-line comment. */

void
example(void)
{
	/* A single-line comment. */
	int		la;

	/* A multi-line comment. */
	int		lb;

	/* A multi-line comment. */
	int		lc;
}
